routes/games.js

```diff
@@ -1,5 +1,6 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const csurfProtection = require('csurf')({ cookie: true });
```

Comment:
What is your thought on creating the configured middleware here as opposed to on it's own line?

Reply:
I felt like since I am not going to do anything else to it, it would be better to configure it on one line.

Reply:
What are the pros/cons of one line?

Reply:
Well I think its a bit subjective. One liners can get long and confusing if alot of stuff is packed into it. However, it can minimize the amount of lines needed and make it more compact. I generally think there is nothing inherently wrong about it as long as its understandable.

Reply:
Good thoughts.

I think when something is subjective it's best to match the documentation of the library and/or the established pattern.

In this case, the established pattern for configurable middleware is to invoke it either inline where the middleware is used or instantiate it on it's own line.

Re: compactness, you articulated a good point that seeking code terseness (short, compact) can cause confusion. You mention that a benefit is it can reduce number of lines.

What is the benefit of less lines?

Reply:
The benefit of having less lines means a smaller code base and efficient in terms of time. Smaller code bases allows other developers that comes onto the project spend less time going through it. And since your not spreading your code through more lines, its a lot quicker to write out than if you were. However, these only matter if your code is readable and easy to maintain.

Reply:
Nice job writing out your thoughts. Writing out what we think gives us the best opportunity to uncover hidden assumptions and understand the boundary between what we know and what we don't know.

We have a few incorrect assumptions in the above.

1. Less lines of code does **not** correlate to efficient code in terms of time.
2. Less lines of code does **not** correlate to decreased time of developer on-boarding.
3. Developers spend the majority of the time **reading** code **not** writing code. Therefore if we want to improve the time it takes a developer to understand a code base we should prioritize self-documenting well written code not the speed of which it takes to write.

I think the core incorrect assumption we are making is that "less lines of code means the code is doing less things."
That is not the case. Less lines of code literally just means less lines of code. Which means more work for a reader of the code to understand it.

The code we write is for humans. The machines will take care of optimizing/compiling it for their consumption.